network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/notification/NetPDisabledNotificationBuilder.kt

```diff
@@ -25,19 +25,21 @@ import android.content.Intent
 import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
 import androidx.core.app.TaskStackBuilder
+import com.com.duckduckgo.networkprotection.subscription.NetpSubscriptionManager
+import com.com.duckduckgo.networkprotection.subscription.isActive
+import com.com.duckduckgo.networkprotection.subscription.isExpired
 import com.duckduckgo.browser.api.ui.BrowserScreens.SettingsScreenNoParams
 import com.duckduckgo.di.scopes.AppScope
 import com.duckduckgo.navigation.api.GlobalActivityStarter
 import com.duckduckgo.networkprotection.impl.R
-import com.duckduckgo.networkprotection.impl.store.NetworkProtectionRepository
 import com.squareup.anvil.annotations.ContributesBinding
 import java.text.DateFormat
 import java.text.SimpleDateFormat
 import java.util.Date
 import javax.inject.Inject
 
 interface NetPDisabledNotificationBuilder {
-    fun buildDisabledNotification(context: Context): Notification
+    suspend fun buildDisabledNotification(context: Context): Notification?
```

Comment:
This can return now `Notification` or `null` because no in all cases we'll show the disabled notification